tests/kas.spec.ts

```diff
@@ -119,3 +119,9 @@ test('test fail to create Kafka instance with the same name', async ({ page }) =
 
   await deleteKafkaInstance(page, testInstanceName);
 });
+
+test('create GCP Kafka instance', async ({ page }) => {
+  await createKafkaInstance(page, testInstanceName, false, null, 'Google Cloud Platform');
+  await waitForKafkaReady(page, testInstanceName);
```

Comment:
do we need this? Isn't it enough just to create it and check that instance is in creating phase? It will save some time

Reply:
added Can I resolve this conversation @Frawless ?